staging/src/k8s.io/client-go/util/workqueue/default_rate_limiters.go

```diff
@@ -236,3 +236,42 @@ func (w WithMaxWaitRateLimiter) Forget(item interface{}) {
 func (w WithMaxWaitRateLimiter) NumRequeues(item interface{}) int {
 	return w.limiter.NumRequeues(item)
 }
+
+var _ RateLimiter = &FakeRateLimiter{}
+
+// FakeControllerRateLimiter is a constructor for a rate limiter for a workqueue.
+func FakeControllerRateLimiter(baseDelay, maxDelay time.Duration, limit int) RateLimiter {
```

Comment:
This wrapper function seems redundant. Can we just use `NewFakeRateLimiter`?

Reply:
Removed the wrapper function